### PR TITLE
Documentation online form

### PR DESCRIPTION
**Pull request by ketankartoza**
This PR is for issue #1287

### CODE REVIEW

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 1 min 51 sec

**Comment by dimasciput** [comment]
@Jeremy-Prior please review

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 1 min 35 sec

File: docs/src/user/manual/data-upload/online-form.md [hunk 1]
```diff
@@ -1,2 +1,168 @@
...
+
+The data upload online form provides a convenient and structured way to enter species population data for your selected property. This form is designed to capture detailed information about species populations, including scientific names, counts, survey methods, and more. In addition, the form displays an image of the selected property in the top left corner, along with property information.
+
+#### Species detail
+
+![Species Detail](./img/online-form-1.png)
+
+1. **Species detail**: First step of data upload online form `SPECIES DETAIL`.
+
+2. **Property detail**: Showing property details alog with the image.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/data-upload/online-form.md:15]:
```suggestion
2. **Property detail**: Showing property details along with the image.
```

File: docs/src/user/manual/data-upload/online-form.md [hunk 2]
```diff
@@ -1,2 +1,168 @@
...
+
+#### Activity Detail
+
+![Activity Detail](./img/online-form-2.png)
+
+1. **Activity Detail**: Second step of data upload online form `ACTIVITY DETAIL`.
+
+2. **Introduction/Reintroduction**: The `Introduction/Reintroduction` section displays the table and form. table for relevant data in a structured tabular format, and  form for entering detailed information related to species introduction or reintroduction.
+
+    ##### Introduction/Reintroduction Table and Form
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/data-upload/online-form.md:83]:
```suggestion
    **Introduction/Reintroduction Table and Form**
```

File: docs/src/user/manual/data-upload/online-form.md [hunk 3]
```diff
@@ -1,2 +1,168 @@
...
+
+    6. **Adult Females**: Specify the count of adult females.
+
+    7. **Juvenile Males**: Provide the count of juvenile males.
+
+    8. **Juvenile Females**: Record the count of juvenile females.
+
+    9. **Total Count**: The total count field will automatically calculate based on the counts entered for adult males, adult females, juvenile males, and juvenile females.
+
+    10. **Founder Population**: Indicate whether the introduced or reintroduced group is considered a founder population. Use the radio buttons to select `Yes` or `No`. This is mendatory.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/data-upload/online-form.md:105]:
```suggestion
    10. **Founder Population**: Indicate whether the introduced or reintroduced group is considered a founder population. Use the radio buttons to select `Yes` or `No`. This is mandatory.
```

File: docs/src/user/manual/data-upload/online-form.md [hunk 4]
```diff
@@ -1,2 +1,168 @@
...
+
+    11. **Event**: Select the appropriate event type from the dropdown menu. Choose the event category that best describes the introduction or reintroduction process. This field is mendatory.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/data-upload/online-form.md:107]:
```suggestion
    11. **Event**: Select the appropriate event type from the dropdown menu. Choose the event category that best describes the introduction or reintroduction process. This field is mandatory.
```

File: docs/src/user/manual/data-upload/online-form.md [hunk 5]
```diff
@@ -1,2 +1,168 @@
...
+
+    12. **Source**: Describe the source of the introduced or reintroduced population. This field is mendatory.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/data-upload/online-form.md:109]:
```suggestion
    12. **Source**: Describe the source of the introduced or reintroduced population. This field is mandatory.
```

File: docs/src/user/manual/data-upload/online-form.md [hunk 6]
```diff
@@ -1,2 +1,168 @@
...
+
+    13. **Permit Number**: Enter the permit number.
+
+    14. **Notes**: Include any additional notes or comments relevant to the introduction or reintroduction event.
+
+    15. **Add**: Click the `ADD` button to save the information entered into the form. This action will record the introduction or reintroduction event with the specified details.
+
+3. **Off-take**: The `Off-take` section displays the table and form. Table provides a clear overview of your off-take data, and form for input data related to species off-take.
+
+    ##### Off-take Table and Form
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/data-upload/online-form.md:119]:
```suggestion
    **Off-take Table and Form**
```

File: docs/src/user/manual/data-upload/online-form.md [hunk 7]
```diff
@@ -1,2 +1,168 @@
...
+
+#### Review & Submit
+
+![Review & Submit](./img/online-form-5.png)
+
+1. **Species Detail**: The `Species Detail` section displays a summary of the species-specific information you've entered, providing an overview of species counts and other relevant details.
+
+2. **Edit Note Icon**: For editing the species detail, simply click on the `Edit Note Icon` to access and modify the species-related information.
+
+
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/data-upload/online-form.md:151]:
```suggestion

```

**Review by Jeremy-Prior — CHANGES_REQUESTED** [review]

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 1 min 54 sec

**Comment by ketankartoza** [comment]
@Jeremy-Prior I have done with all the suggested changes. Please review. Thanks

**Review by Jeremy-Prior — APPROVED** [review]
LGTM

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 2 min 23 sec

**Pull request merged**